scripts/did.sh

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 
 function generate_did() {
-  local canister=$1
-  local candid_file="$(canister="$canister" jq -r .canisters[env.canister].candid dfx.json)"
+  local canister candid_file
+  canister=$1
+  candid_file="$(canister="$canister" jq -r '.canisters[env.canister].candid' dfx.json)"
```

Comment:
This fixes a lint issue (local declaration and assignment should be separate, and the jq argument should be quoted).  This should not have got through CI into main, so we have a gap there.